zarr/meta.py

```diff
@@ -459,6 +463,36 @@ def _decode_codec_metadata(cls, meta: Optional[Mapping]) -> Optional[Codec]:
 
         return codec
 
+    @classmethod
+    def _encode_storage_transformer_metadata(
```

Comment:
After the splitting of `storage.py` into separate files, this makes me wonder if meta and some of the other core classes shouldn't also have v3 variants.